core/src/main/java/org/ehcache/core/spi/cache/AbstractValueHolder.java

```diff
@@ -109,7 +124,16 @@ public long lastAccessTime(TimeUnit unit) {
   }
 
   public void setLastAccessTime(long lastAccessTime, TimeUnit unit) {
-    this.lastAccessTime = unit.convert(lastAccessTime, nativeTimeUnit());
+    lastAccessTime = unit.convert(lastAccessTime, nativeTimeUnit());
```

Comment:
hehe scratch that, you are assigning to the param. Well, could we have a local var with a name instead? This is confusing with the name shadowing in place already.